### ISSUE TITLE
Add error log for the application

### ISSUE THREAD

**Issue by mauriciovieira** [apinf/platform#638]
Copied from #615 

# Description

There should be a centralized way to log server errors like the one reported at #636 

# User story

```
As a system administrator
I would like to review log files for the Apinf platform
so that I can identify issues, warnings, etc. and take corrective action
```
# Feature

Research and choose a logging framework for Apinf (Meteor.js). Add the logging framework/package. Determine what functions, e.g. server methods, need logging support and add calls to the logger with relevant details.

Once logging is enabled, it will be useful to log errors (e.g. within try/catch statements).
# Research

The Winston logging library may be a good solution:
- [Meteor Winston](https://atmospherejs.com/votercircle/winston)
- [Winston Client](https://atmospherejs.com/brentjanderson/winston-client)
(to be checked)

# Related
- Issue #636 describes error handling within the `syncApiBackends` method.
- Issue #638 purposes adding an error log to the aplication

## Definition of done
- [ ] Log should exist as file on server.
- [ ] Any error handling should be update to log.
- [ ] Log files should be rotated, i.e. should not grow indefinitely.
- [ ] Log items should contain severity level.
- [ ] Log items should contain time stamp.
- [ ] Optionally, log items should be accessible through administrative page.
- [ ] Project should use existing logging framework/package, if possible.

**Comment by mauriciovieira** [comment]
I would like to work on this task

**Comment by brylie** [comment]
@mauricio, will you please estimate this task size?

**Comment by brylie** [comment]
Please estimate the size of this task. :-)

**Comment by mauriciovieira** [comment]
closed by #1986
